src/components/Login/RegisterComponent.js

```diff
@@ -30,6 +30,38 @@ export default function RegisterComponent() {
     return randomEmail;
   };
 
+  const generateProgressTables = async (u) => {
+    const query = new Parse.Query("Category");
+    let categories = await query.find();
+
+    for (let i = 0; i < categories.length; i++) {
+      const progressTable = new Parse.Object("Progress");
+      progressTable.set("user_id", u.id);
+      progressTable.set("category_name", categories[i].get("name"));
+      await progressTable.save();
+    }
+    /*const number = new Parse.Object("Category");
```

Comment:
large comment. I do not know if you need it for later, so just wanted to make a note, in case it needs to be deleted :)

Reply:
Will delete it